app/_hub/kong-inc/acme/_index.md

```diff
@@ -231,6 +238,44 @@ plugins:
   - name: acme
     config:
       account_email: example@myexample.com
+      account_key:
+        key_id: "1234"
```

Comment:
we might also want to include the section that actually defines this key_id and key_set in dbless config.

Reply:
Is there an example of the `keys` and `keys_sets` being defined for declarative config? I've tried looking around and haven't found any.

My guess is it looks something like the following but I'm not 100% sure
```yaml
key_sets:
  name: example-key-set
keys:
  example-key:
    set: example-key-set
    pem:
      private_key: {vault://env/example-private-key}
```

Reply:
Updated the config in my latest commit